### DISCUSSION TITLE
Deploying to Linode Ubuntu 20.04 LTS

### DISCUSSION THREAD

**Discussion by Alii-isk** [pocketbase/pocketbase#908] [Q&A]
hello, please i am trying to deploy pocket base to  Ubuntu 20.04 LTS on linode .
i followed the guide at this url https://pocketbase.io/docs/going-to-production/ but i cannot access it on the web

```
  root@localhost:~./pocketbase serve --http="SERVER_IP:80" --https="SERVER_IP:443"
  > Server started at: https://SERVER_IP:443
    - REST API: https://SERVER_IP:443/api/
    - Admin UI: https://SERVER_IP:443/_/
  2022/11/03 00:22:56 http: TLS handshake error from SERVER_IP:61416: acme/autocert: missing server name
  2022/11/03 00:22:56 http: TLS handshake error from SERVER_IP:61417: acme/autocert: missing server name
  2022/11/03 00:22:56 http: TLS handshake error from SERVER_IP:61418: acme/autocert: missing server name
  2022/11/03 00:22:56 http: TLS handshake error from SERVER_IP:61419: acme/autocert: missing server name
```
I am not very knowledge with servers if someone could help me i would be very  appreciated

**Comment by MSSArtK0d3r** [comment]
Hello Alii-isk,
Change SERVER_IP to your linode IP address
example:
./pocketbase serve --http="10.132.32.16:80" --https="10.132.32.16:443"

which 10.132.32.16 is your VPS IP address

Hope this helps

**Reply by Alii-isk** [reply]
thank you for replay,but i already did that i am just masking it here for security reasons

**Comment by ganigeorgiev** [comment]
@Alii-isk, @MSSArtK0d3r "Let's encrypt" doesn't allow issuing certificates on bare IPs. If you want to have https you'll have to use a valid public domain name - https://community.letsencrypt.org/t/certificates-for-ip-addresses/158093/5, aka:

```sh
./pocketbase serve --http="yourdomain.com:80" --https="yourdomain.com:443"
```
